src/core_modules/capture-core/components/Pages/Search/SearchPage.component.js

```diff
@@ -80,20 +121,69 @@ const Index = ({ classes, trackedEntityTypesWithCorrelatedPrograms, preselectedP
                                                     value={trackedEntityTypeId}
                                                     label={trackedEntityTypeName}
                                                 />,
-                                                programs.map(({ programName, programId }) =>
+                                                tePrograms.map(({ programName, programId }) =>
                                                     (<SingleSelectOption value={programId} label={programName} />)),
-                                                <hr key={trackedEntityTypeId} />,
+                                                <div className={classes.divider} key={trackedEntityTypeId}>
+                                                    <hr />
+                                                </div>,
                                             ],
                                         ),
                                     [
+                                        classes.divider,
                                         trackedEntityTypesWithCorrelatedPrograms,
                                     ])
                                 }
                             </SingleSelect>
                         </div>
                     </div>
                 </Section>
+
+                {
```

Comment:
Maybe extract the unique search part into its own component to make it a bit easier on the eyes? (Also applies for the SearchDomainSelector related code)

Reply:
yeah valid and I agree. Both are happening in later PRs. You will see it :)